### PR TITLE
Fix for missing TokenFragment class

### PR DESCRIPTION
**Pull request by Noah1911**
Resolves "ERROR: Exception launching auth code request: java.lang.ClassNotFoundException: com.google.games.bridge.TokenFragment" - looks like it is executed from wrong thread (fixes use of this method in other threads).

ref #2397

### CODE REVIEW

File: source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs [hunk 1]
```diff
@@ -341,8 +341,8 @@ public string GetServerAuthCode()
         public void GetAnotherServerAuthCode(bool reAuthenticateIfNeeded,
                                              Action<string> callback)
         {
-            mTokenClient.GetAnotherServerAuthCode(reAuthenticateIfNeeded,
-                                                  callback);
+            PlayGamesHelperObject.RunOnGameThread(() => mTokenClient.GetAnotherServerAuthCode(reAuthenticateIfNeeded,
```
Comment (olehkuznetsov) [inline source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs]: It seems we have similar code in AndroidTokenClient.cs for FetchTokens
```
        public void FetchTokens(bool silent, Action<int> callback)
        {
            PlayGamesHelperObject.RunOnGameThread(() => DoFetchToken(silent, callback));
        }

        internal void DoFetchToken(bool silent, Action<int> callback)
        { ... }
```
Probably, we should have there the same logic for 'GetAnotherServerAuthCode'. I mean adding
`internal void DoGetAnotherServerAuthCode(bool silent, Action<int> callback)`
and calling it using `PlayGamesHelperObject.RunOnGameThread`

**Comment by Noah1911** [comment]
Forced pushed it due to messed stuff, now it should be okay.

**Comment by olehkuznetsov** [comment]
Thanks a lot.

**Comment by tinkingli** [comment]
hi,I also encountered the same problem. I hope you can help me.

**Comment by olehkuznetsov** [comment]
Try v0.9.58. Should be fixed there.

**Comment by tinkingli** [comment]
Project, all with v0.9.58?

**Comment by tinkingli** [comment]
I tried it again, or not.

**Comment by tinkingli** [comment]
No Running .......

**Comment by Noah1911** [comment]
@tinkingli open new issue with error and proper stack trace

**Comment by tinkingli** [comment]
I have the same problem“ERROR: Exception launching auth code request: java.lang.ClassNotFoundException: com.google.games.bridge.TokenFragment”unity2017.4.1

**Comment by tinkingli** [comment]
I have the same problem“ERROR: Exception launching auth code request: java.lang.ClassNotFoundException: com.google.games.bridge.TokenFragment”
unity version  2017.4.1

**Comment by tinkingli** [comment]
On the phone, click on the login will appear this error

**Comment by tinkingli** [comment]
ok,thank you

**Review by mega123-st — APPROVED** [review]

**Pull request merged**